### ISSUE TITLE
Wrong translation key in aria-label="Visionouvelle fenêtre"

### ISSUE THREAD

**Issue by virgile-dev** [suitenumerique/docs#2019]
## Bug Report

**Problematic behavior**
The translation key for the aria-label content is not right. It was added in https://github.com/suitenumerique/docs/pull/1984

Curently we have this `aria-label="Visionouvelle fenêtre"`
The translation key is set like this `newWindowLabelSuffix={t('new window')}`

**Expected behavior/code**
We should have `aria-label="Visio (nouvelle fenêtre)"`

The translation key should be  `newWindowLabelSuffix={t(' (new window)')}
I included the space between Visio and (nouvelle fenêtre) . I don't know if it's a good practice.

**Steps to Reproduce**
1. Do this...
2. Then this...
3. And then the bug happens!

**Environment**
- Docs version: 4.8
- Instance url: docs.numerique.gouv.fr

**Possible Solution**

**Additional context/Screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by Ovgodd** [comment]
Hi @virgile-dev , sorry about that. 
I assumed it had to be done in crowdin which is why I left it as a simple key. 
I’ll change the key as you suggested
